### PR TITLE
Feat wishlist

### PR DESCRIPTION
**Pull request by ankit9015**
This PR has following features:
1. Wishlist functionality added
2. Header, ProductCard, CartCard are updated to support wishlist functionality

### CODE REVIEW

**Comment by atharvabhanage02** [comment]
Preview isn't working ?

**Comment by ajit-sharmaGH** [comment]
> Preview isn't working ?

it's working

File: src/App.js [hunk 1]
```diff
@@ -5,20 +5,23 @@ import { ProductProvider } from "./context/ProductContext/ProductContext";
...
-            <div className="page-layout__main">
-              <Router />
+        <WishlistProvider>
+          <ProductProvider>
+            <div className="page-layout">
+              <Header></Header>
+              <div className="page-layout__main m-l">
+                <Router />
+              </div>
+              <Footer></Footer>
```
Comment (ajit-sharmaGH) [inline src/App.js:21]: If you are not using children inside footer you can simply use self closing tag of this footer component like `<Footer />`

**Review by ajit-sharmaGH — APPROVED** [review]
Looks good clean and readable code
You can merge it

**Pull request merged**